tests/test_run_haplotag.py

```diff
@@ -111,6 +111,578 @@ def test_haplotag_cli_parser(tmp_path):
     assert ps_count > 0
 
 
+def test_haplotag_cli_parser_supplementary_strategy_no_flag(tmp_path):
```

Comment:
This is a lot of (duplicated) code that essentially only tests whether ArgumentParser was used correctly. Please at least merge this and the next two functions into one. You can test all the options by using `pytest.mark.parametrize`.

Reply:
fixed